### DISCUSSION TITLE
Scale the pgsync on kubernates

### DISCUSSION THREAD

**Discussion by jayantwaghulde** [toluaina/pgsync#298] [General]
Documentation says pgsync can scale on demand.
We are running pgsync on kubernetes. Does that mean I can have multiple pods of pgsync running in my cluster?
If this is not the case then how can we scale the pgsync, can you please guide us?

**Comment by johija** [comment]
Hi, looking at PGsync to replace Logstash due to slowness while handling huge amount of data. Can someone answer this query?

**Comment by toluaina** [comment]
You cannot at present run multiple instances of PGSync. This makes transnational consistency hard to achieve.
You can however tune the number of threads at runtime as well as using the async mode. 
Postgres scales vertically and Elasticsearch scales horizontally. The event driven operational model of PGSync ensures that you are able to process large volumes of data continuously.

**Reply by johija** [reply]
Thanks @toluaina
By async mode do you mean the daemon mode?
